.github/workflows/trigger-ui-server-update.yml

```diff
@@ -0,0 +1,26 @@
+name: 'Trigger ui-server Update'
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  notify:
+    name: 'trigger ui-server update'
+    runs-on: ubuntu-latest
+
+    defaults:
+      run:
+        shell: bash
+
+    steps:
+      - name: Dispatch ui-server Github Action
+        env:
+          PAT: ${{ secrets.PAT }}
+          PARENT_REPO: temporalio/ui-server
+          PARENT_BRANCH: main
+          WORKFLOW_ID: update-ui.yml
+        run: |
+          curl -fL -X POST -H "Accept: application/vnd.github.v3+json" -H "Authorization: token ${{ env.PAT }}" https://api.github.com/repos/${{ env.PARENT_REPO }}/actions/workflows/${{ env.WORKFLOW_ID }}/dispatches -d '{"ref":"${{ env.PARENT_BRANCH }}"}'
```

Comment:
I imagine that there is probably away to abstract this into a re-usable action—but I think this totally works for now.

Reply:
that's true. I've looked at few github actions available in the marketplace that do this instead of manually calling `curl`, though i was a bit hesitant to use them because of the added security implication and having to rely on a third-party lib. Figured to have a direct REST call instead

Still open to the idea though